releases/release-6.2.0.md

```diff
@@ -0,0 +1,450 @@
+
+* TiDB supports concurrent DDL operations
+
+    TiDB v6.2.0 introduces a new concurrent DDL framework, which enables DDL statements to be concurrently executed on different table objects and fixes the issue that DDL operations are blocked by DDL operations on other tables. In addition, TiDB supports concurrent DDL execution when adding an index on multiple tables or changing a column type.This improves the efficiency of DDL execution.
+
+    [User document](/system-variables.md#tidb_enable_concurrent_ddl-new-in-v620) [#32031](https://github.com/pingcap/tidb/issues/32031) @[wjhuang2016](https://github.com/wjhuang2016)
+
+* Optimizer enhances the estimation of string matching
+
+    In the string matching scenario, if the optimizer cannot accurately estimate the number of rows, it affects the generation of the optimal execution plan. For example, the condition is `like '%xyz'` or using a regular expression `regex ()`. To improve the estimation accuracy in such scenarios, TiDB v6.2.0 enhances the estimation method. The new method combines the TopN information of statistics and system variables to improve the accuracy and makes it possible to modify manually, thus improving the SQL performance.
+
+    [User document](/system-variables.md#tidb_default_string_match_selectivity-new-in-v620) [#36209](https://github.com/pingcap/tidb/issues/36209) @[time-and-fate](https://github.com/time-and-fate)
+
+* Window functions pushed down to TiFlash can be executed in multiple threads
+
+    By enabling the Fine Grained Shuffle feature, the window function can be executed from in a single thread to in multiple threads. It reduces the query response time significantly. This feature does not change user behavior. You can control the granularity of the shuffle by adjusting the value of the variables.
+
+    [User document](/system-variables.md#tiflash_fine_grained_shuffle_batch_size-new-in-v620) [#4631](https://github.com/pingcap/tiflash/issues/4631) @[guo-shaoge](https://github.com/guo-shaoge)
+
+* TiFlash supports a newer version of storage format
+
+    The new storage format relieves high CPU usage caused by GC in high-concurrency and heavy workload scenarios. This significantly reduces IO traffic of background tasks, thereby boosting stability under high concurrencies and heavy workloads. At the same time, space amplification and disk waste can be significantly reduced.
+
+    In TiDB v6.2.0, data is stored in the new storage format by default. When you upgrade TiFlash from earlier versions to v6.2.0, existing data inherits the previous storage format, and newly written data is stored in the new format. Note that if TiFlash is upgraded from earlier versions to v6.2.0 and has received new data, you cannot perform in-place downgrade on TiFlash, because earlier TiFlash versions cannot recognize the new storage format.
```

Comment:
```suggestion
    In TiDB v6.2.0, data is stored in the new storage format by default. Note that if TiFlash is upgraded from earlier versions to v6.2.0, you cannot perform in-place downgrade on TiFlash, because earlier TiFlash versions cannot recognize the new storage format.
```